tests/integration-tests/tests/arm_pl/test_arm_pl/test_arm_pl/pcluster.config.yaml

```diff
@@ -6,6 +6,8 @@ HeadNode:
     SubnetId: {{ public_subnet_id }}
   Ssh:
     KeyName: {{ key_name }}
+  Imds:
+    Secured: {{ imds_secured }}
```

Comment:
why did this need to be added? we can't rely on the default for all but the Batch tests?

Reply:
Sure, but I think that as long as we have scheduler as a parameterized config in config file, the same should be done for imds secured because we do not know apriori anything about the scheduler. An alternative approach may be to put a jinja if statement within the config file, but I think this would be conceptually incorrect since config values should be injected by fixtures. 

What do you think?